doc/architecture-design.md

```diff
@@ -643,13 +645,13 @@ Enable advanced debug mode by modifying the configuration in `conf/debug.yaml` f
 
 The checker would judge whether the file data changed according to the last modification time of the file. If there has any change, reload it. If there was no change, skip this check. So it's hot reload for enabling or disabling advanced debug mode.
 
-|Key|Optional|Description|Default|
-|----|-----|---------|---|
-|hook_conf.enable|required|Enable/Disable hook debug trace. Target module function's input arguments or returned value would be printed once this option is enabled.|false|
-|hook_conf.name|required|The module list name of hook which has enabled debug trace||
-|hook_conf.log_level|required|Logging levels for input arguments & returned value|warn|
-|hook_conf.is_print_input_args|required|Enable/Disable input arguments print|true|
-|hook_conf.is_print_return_value|required|Enable/Disable returned value print|true|
+| Key                             | Optional | Description                                                                                                                               | Default |
+| ------------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| hook_conf.enable                | required | Enable/Disable hook debug trace. Target module function's input arguments or returned value would be printed once this option is enabled. | false   |
+| hook_conf.name                  | required | The module list name of hook which has enabled debug trace                                                                                |         |
+| hook_conf.log_level             | required | Logging levels for input arguments & returned value                                                                                       | warn    |
+| hook_conf.is_print_input_args   | required | Enable/Disable input arguments print                                                                                                      | true    |
+| hook_conf.is_print_return_value | required | Enable/Disable returned value print                                                                                                       | true    |
```

Comment:
I think it would be better to add `.` after the sentence in the `Description` field.

Reply:
Fixed. Thanks!